protocol/client/event.go

```diff
@@ -16,6 +16,16 @@ const (
 	EventTypeError
 )
 
+// Event is used to workaround event.Feed type checking.
```

Comment:
This comment sums up why I don't like `github.com/ethereum/go-ethereum/event` :D

Reply:
the problem is that interface type checking is not working/implemented. if channel is using value types then everything works as expected.

i think that can be added by checking if feed.etype is interface and then checking that sent value implements that interface..

but in general if there is a better implementation for one-to-many(or none) messaging i can use it

Reply:
I don't know anything better now. Let's keep it.